### ISSUE TITLE
Bookmark button usability

### ISSUE THREAD

**Issue by smonff** [tuskyapp/Tusky#1677]
Just installed Tusky 10.0 beta 1.

This is only a personal feeling, but I feel like the new *bookmark* button is misplaced because it is actually where was the *fav* button before. When I want to fav a top, I try to press *bookmark*. Note I am using my right hand and that I didn't used TuskyX long enough to be able to change my habits.

Maybe the bookmark button could be on the left of the fav?

Just some user experience issue.
* * * *
- Tusky Version: 10.0 beta 1
- Android Version: 9
- Android Device: Samsung A6
- Mastodon instance (if applicable): https://cyber.space

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.

**Comment by connyduck** [comment]
I'm thinking about moving the button into the menu like the Mastodon interface web did.

**Comment by ariasuni** [comment]
Mastodon Web displays a bookmark button but only in the expanded view of a toot (i.e. when the toot is opened in its own column). It might a good idea to do the same thing.

**Comment by TheOtherDoctor** [comment]
I understand that you consider having the bookmark button not accessible in the normal list view - that I'd see as opposing the (my) main usage of the bookmark button as "remember to follow up on this later" which is how I mostly use it - like, there is a toot with some links I'd want to have a look later. Having bookmark hidden in a deeper menu would mess up such a workflow, thus I'd highly prefer to have it directly available.

When it comes to upcoming repositioning of the buttons below a toot, please consider moving the rightmost one far enough to the left to assert it is not taken as a "tap on the right border" which seems to be triggering the "open toot detail" action? I typically need several trials to hit the rightmost button as it always ends up opening the toot. :-(

**Comment by charlag** [comment]
@connyduck I agree, I think it might be too prominent to be in the list (it's a rather rare action)

**Comment by charlag** [comment]
sounds good to me

**Comment by Tak** [comment]
Do we still want to move this into the menu?
